config/samples/catalog/catalog.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
I don't think this should go in `config/samples/catalog` what about in it's own config directory? i.e `config/catalog`

Reply:
I just used the same layout we used in NOO but since this just for dev not sure if we should expose outside samples but I am fine either way, do u have motivation for it being under config ?

Reply:
No I don't care that much I just think isn't a sample at all, you could even put it under `hack/` since really it's just a hack to preview the developer in operatorhub

Otherwise I don't care too much, if you want to leave it here that's fine

Reply:
I moved out under config